core/src/main/java/org/elasticsearch/action/search/TransportSearchAction.java

```diff
@@ -284,8 +285,9 @@ private void executeSearch(SearchTask task, SearchTimeProvider timeProvider, Sea
         for (int i = 0; i < indices.length; i++) {
             concreteIndices[i] = indices[i].getName();
         }
+        Map<String, Long> nodeSearchCounts = searchTransportService.getPendingRequests(SearchAction.NAME);
```

Comment:
You're right! I think instead it should be the prefix of "indices:data/read/search" so that all of [these](https://github.com/dakrone/elasticsearch/blob/bbeda3c59f133e9ccabaea5cebebc4d0a5fa6df8/core/src/main/java/org/elasticsearch/action/search/SearchTransportService.java#L70-L80) match it.

I'll make that change and then re-run the benchmarks to see if it affects things, it may mean we can lower `b=4` back down to `b=3`, good catch!